questions/po_tools.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: "PO tools"
+parent: Katas
+---
+Which tools are mandatory for the Product Owner?
```

Comment:
I would either refine or reject this question for two reasons:
- I believe there can't be anything "mandatory" for the PO;
- The question doesn't contain a challenge and serves the purpose of knowledge/experience sharing in a group.

Reply:
To clarify my comment I bring a similar question: What are the ways to hold Daily Scrum effective?

It's also more about making a suggestion. Should we have such questions? 

If we decide that we should, I'd suggest to refine the question in the following way:

What tools might help PO to {{do something}} effectively?

Reply:
For me it's also too generic. I also would not add/change the question ourselves, as unlike #51 it's hard to to see the intent of the question (at least for me). We could reach out to Manolo to check on this. But I'd also say that like this we should not inlclude it.

Reply:
I'll reach out to Manolo on Monday and ask him to elaborate on the question's intent.